infrastructure/terraform/critical/api.wellcomecollection.org/variables.tf

```diff
@@ -9,3 +9,5 @@ variable "cert_domain" {
 variable "public_api_bucket_domain_name" {}
 
 variable "description" {}
+
+variable "cf_logging_bucket" {}
```

Comment:
Call it `cloudfront_logging_bucket` so it makes more sense for the next person who reads this code?